snapcraft/linters/library_linter.py

```diff
@@ -101,6 +104,20 @@ def run(self) -> List[LinterIssue]:
 
         return issues
 
+    def _find_deb_package(self, library_name):
```

Comment:
this is missing types 
```suggestion
    def _find_deb_package(self, library_name: str) -> Optional[str]:
```